continuous-integration/setup-guidelines.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I am a bit confused by this article mostly because I am not sure if this is the correct place. 
If we decide to let it be here it should provide some kind of summary of what the rest of the articles in this section will hold.